spec/javascripts/govuk_publishing_components/analytics/explicit-cross-domain-links.spec.js

```diff
@@ -47,7 +47,7 @@ describe('Explicit cross-domain linker', function () {
       element = $('<a href="/somewhere">')
     })
 
-    it('modifies the link href to append cookie_consent parameter "not-engaged" if cookie preferences are not set', function () {
+    it('modifies the link href to append cookie_consent parameter "not-engaged" if cookies_preferences_set cookie is not "true"', function () {
```

Comment:
Can we have a test for the new case?  Presumably this catches the case where there is no `cookie_preferences_set` cookie

Reply:
Thanks, I've updated the spec